.actions/assistant.py

```diff
@@ -60,6 +60,23 @@ def _load_py_module(name: str, location: str) -> ModuleType:
     return py
 
 
+def _retrieve_files(directory: str, *ext: str):
+    subfolders, files = [directory], []
+
+    ext = tuple(e.lower() for e in ext)
+
+    while subfolders:
+        for p in os.scandir(directory):
+            if os.path.isdir(p):
+                subfolders.append(p)
+
+            if os.path.isfile(p):
+                if not ext or os.path.split(p.name)[1].lower() in ext:
```

Comment:
this should avoid possible problems with `.pyc` files matching `.py`

```suggestion
                if not ext or any(os.path.split(p.name)[1].lower().endswith(e) for e in ext):
```